source/logger.js

```diff
@@ -0,0 +1,60 @@
+if(Meteor.isServer)
+  winston = Npm.require('winston');
+
+Space.Logger = Space.Object.extend(Space, 'Logger', {
+
+  _logger: null,
+  _state: 'stopped',
+
+  Constructor: function(){
+
+    if(Meteor.isServer) {
+      this._logger = new (winston.Logger)({
+        transports: [
+          new (winston.transports.Console)()
+        ]
+      });
+    }
+    if(Meteor.isClient) {
+      this._logger = console
+    }
+  },
```

Comment:
We have two logger instances being setup, so the Space.log needs to stay as is to be available outside the scope of a Module, but the app-level logger can probably just be mixed in to Space.Application which would clean up that extra stuff I've introduced into Module